src/salesforce-config.ts

```diff
@@ -40,19 +40,19 @@ export async function readSalesforceConfig(
   let salesforceApiVersion = salesforceConfig?.["salesforce-api-version"];
   if (!salesforceApiVersion) {
     console.log(
-      "DEPRECATION NOTICE: com.salesforce.salesforce-api-version is not defined in project.toml and has been defaulted to '53.0'. This field will be required in a future release."
+      "DEPRECATION NOTICE: com.salesforce.salesforce-api-version is not defined in project.toml and has been defaulted to '55.0'. This field will be required in a future release."
     );
-    salesforceApiVersion = "53.0";
+    salesforceApiVersion = "55.0";
   }
 
   if (
     !semver.satisfies(
       semver.coerce(salesforceApiVersion),
-      `>=${semver.coerce("53.0")}`
+      `>=${semver.coerce("55.0")}`
```

Comment:
the SDK does support `v53.0` though, right?  is this check just to verify the minimum version allowed?  because, if so, it should probably stay at `53.0`

Reply:
ahh fair, let me adjust this